### ISSUE TITLE
Third row for cybran economy UI

### ISSUE THREAD

**Issue by Garanas** [FAForever/fa#3394]
![image](https://user-images.githubusercontent.com/15778155/131308283-226e1a7b-3663-4e4a-8e6f-1dc1fba4912a.png)

The image speaks for itself. This only happens on FAFDevelop.

**Comment by Garanas** [comment]
I can only reproduce this when it is launched through the client.

**Comment by Garanas** [comment]
Launched through client with develop branch:
![image](https://user-images.githubusercontent.com/15778155/131311310-92e9e91d-942a-4bc7-b5d6-7d5a2ab394f0.png)

Launched locally with (latest) develop branch:
![image](https://user-images.githubusercontent.com/15778155/131311370-ef2bb429-a00a-4c59-af31-097b49ab9401.png)

What is the difference? The code that is being run should be the same. What am I missing here?

**Comment by KionX** [comment]
Not reproduce anyway.
This is a different texture. Maybe mods, others .nx5

**Comment by Garanas** [comment]
Several people have had this issue - I made sure there were no mods on when I tried it myself.

**Comment by Tagada14** [comment]
I can reproduce it as well.

**Comment by KionX** [comment]
Maybe this will help you:
UI_Lua WARN(repr(import('/lua/ui/game/economy.lua').GUI.bg.panel._filename))

**Comment by Garanas** [comment]
![image](https://user-images.githubusercontent.com/15778155/131336308-4fe90b07-ed86-4fb0-affe-c9a2ec730a23.png)

Which is a file part of the base game.

![image](https://user-images.githubusercontent.com/15778155/131336376-e27da845-f09a-4867-95c7-c51602d8f2e5.png)

Which appears to be just fine.

**Comment by KionX** [comment]
I believe that this means that the code has not been changed.
It looks like it's in some kind of archive.

**Comment by Garanas** [comment]
Those textures are not part of the repo:

![image](https://user-images.githubusercontent.com/15778155/131337519-a7a434e1-fe0b-4654-af0c-52706e114357.png)

But it has to come from somewhere.

**Comment by KionX** [comment]
whitelist in init_faf. lua and init_fafdevelop.lua is different.
init_fafdevelop. lua contains a non-existent file props.nx5 instead of modules.nx5

**Comment by Garanas** [comment]
I don't think that is it. The modules folder is essentially gaz_ui, which has a .nxt2 and .nxt5 version on my system. Essentially identical.

**Comment by Garanas** [comment]
I'll do a binary search tonight to find the PR that did this.

**Comment by Garanas** [comment]
Yes - it was due to a mod in the era where we allowed mods to load in any textures 😄
